book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_reader :rentals
+  attr_accessor :title, :author
```

Comment:
- rentals have both Getters and setters methods (`attr_reader` and `attr_writer` respectively ). Therefore remove it to where it is and move it to `attr_accessor`. 
-[ Read More here](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/articles/oop_principles_examples.md#useful-shortcuts-attr_reader-attr_writer-and-attr_accessor)